smoke.c

```diff
@@ -98,6 +99,22 @@ void print_array(const TI_REAL *a, int size) {
 
 /*********** PARSING, TESTING, REPORTING ************/
 
+int equal_answers(const ti_indicator_info *info, TI_REAL *answers[], TI_REAL *outputs[], int answer_size, int output_size) {
```

Comment:
it seems to me that `compare_answers` would be a better name, but overall, this refactoring was needed, thanks

Reply:
Haha, I think I had called it that originally, but changed it to this to be consistent with your `equal_arrays` function name. I'll update it.